ckanext/canada/helpers.py

```diff
@@ -58,6 +58,12 @@
 RELEASE_DATE_FACET_STEP = 100
 
 
+def is_registry_domain() -> bool:
+    uri_parts = urlsplit(request.url)
```

Comment:
If some command from the CLI or a background job with no associated request ends up hitting this helper what happens?

Reply:
also tests. We wouldn't need all those mocks added.

Reply:
k catching RuntimeError and returning True for CLI and background processes, added more comments in there